src/pages/benefits/viewAllBenefit/ViewAllBenefits.tsx

```
@@ -77,88 +53,79 @@ const ActionCell = ({ rowData }: ICellTextProps) => {
-
-    const response = await viewAllBenefitsData(payload);
+  const [pageIndex, setPageIndex] = useState(0);
 
-    if (response) {
-      setData(response);
+  const fetchAllBenefits = async () => {
+    try {
+      const response = await getBenefitList();
+      console.log("Response from API:", response);
+      if (response?.results) {
+        const mappedData = response.results.map((benefit: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)